service-base/src/main/java/org/eclipse/hono/service/registration/BaseRegistrationService.java

```diff
@@ -61,9 +62,19 @@
 
     /**
      * The name of the field in a device's registration information that contains
-     * the identifier of the gateway that it is connected to.
+     * the identifier of the gateway that it is connected to (either as string value or inside a JSON array). 
      */
     public static final String PROPERTY_VIA = "via";
+    /**
+     * The name of the field in a device's registration information that contains
+     * the identifier of the gateway that it was last connected to as well as the date when this information was updated.
+     */
+    public static final String PROPERTY_LAST_VIA = "last-via";
+    /**
+     * The name of the field in a device's registration information that contains
+     * the date when the 'last-via' device id was last updated.
+     */
+    public static final String PROPERTY_UPDATE_DATE = "update-date";
```

Comment:
Wouldn't it be better to name the variable so that it is clear that this update_date is meta info about the last-via property? I'm not good in naming but maybe something like: PROPERTY_UPDATE_DATE_LAST_VIA?

Reply:
yes, right, that also fits better to the javadoc content. I've named it PROPERTY_LAST_VIA_UPDATE_DATE now.